fuse/pos_node.go

```diff
@@ -39,6 +40,11 @@ func (n *PosNode) Attr(ctx context.Context, attr *fuse.Attr) error {
 	attr.Uid = uint32(n.fsys.Uid)
 	attr.Gid = uint32(n.fsys.Gid)
 	attr.Valid = 0
+
+	if lt := n.db.LastTouch(); !lt.Equal(time.UnixMilli(0)) {
+		attr.Mtime = lt
+	}
```

Comment:
I guess `CommitWAL` and `CommitJournal` could set this to `-1` or something to indicate that the mtime should always be the current timestamp. We could set `attr.Valid = time.Millisecond`.

Reply:
You can pull the timestamp from the last LTX file applied. If it's always -1 then wouldn't it seem like it's always updating?

As for `attr.Valid`, I think leaving it as `0` (from above) is fine. Caching for a millisecond doesn't seem like it'd help most of the time as I don't think anyone will be checking the replication lag every millisecond.